src/Approvals/SFA.DAS.Approvals.Api/Controllers/CohortController.cs

```diff
@@ -150,6 +151,30 @@ public async Task<IActionResult> GetAddDraftApprenticeshipCourse([FromQuery] lon
             }
         }
 
+        [HttpGet]
+        [Route("employer/{accountId}/unapproved/add/confirm-employer")]
+        [Route("provider/{providerId}/unapproved/add/confirm-employer")]
+        public async Task<IActionResult> GetHasDeclaredStandards([FromQuery] long? providerId)
```

Comment:
I don't think you need the providerId parameter here since you have ServiceParameters in the handler itself.